plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/security/KafkaSecurityConfig.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.security;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import org.apache.kafka.common.config.SslConfigs;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ *  {@SecurityConfig} manages Kafka SSL authentication and encryption between clients and brokers.
+ */
+public class KafkaSecurityConfig
```

Comment:
It is all about SSL, so let's name it `KafkaSslConfig`. I don't think this config will hold any authentication (like SCRUM or PLAIN) properties.

Reply:
Got it. I actually planned on using this class to store SASL SCRAM, but we can use a dedicated class for it.

Reply:
Hmm... Actually, there will be `SASL_SSL`. Should we keep this as `KafkaSslConfig` in this case?

Reply:
> Should we keep this as KafkaSslConfig in this case?

Yes.